test/timearray.jl

```diff
@@ -528,44 +527,65 @@ end
 │ 2000-01-10 │ 102.0  │ 102.25 │ 94.75  │ 97.75  │
 │ 2000-01-11 │ 95.94  │ 99.38  │ 90.5   │ 92.75  │
 │ 2000-01-12 │ 95.0   │ 95.5   │ 86.5   │ 87.19  │
-│ 2000-01-13 │ 94.48  │ 98.75  │ 92.5   │ 96.75  │
-   ⋮
-│ 2001-12-19 │ 20.58  │ 21.68  │ 20.47  │ 21.62  │
+│ ⋮          │ ⋮      │ ⋮      │ ⋮      │ ⋮      │
 │ 2001-12-20 │ 21.4   │ 21.47  │ 20.62  │ 20.67  │
 │ 2001-12-21 │ 21.01  │ 21.54  │ 20.8   │ 21.0   │
 │ 2001-12-24 │ 20.9   │ 21.45  │ 20.9   │ 21.36  │
 │ 2001-12-26 │ 21.35  │ 22.3   │ 21.14  │ 21.49  │
 │ 2001-12-27 │ 21.58  │ 22.25  │ 21.58  │ 22.07  │
 │ 2001-12-28 │ 21.97  │ 23.0   │ 21.96  │ 22.43  │
-│ 2001-12-31 │ 22.51  │ 22.66  │ 21.83  │ 21.9   │"""
+│ 2001-12-31 │ 22.51  │ 22.66  │ 21.83  │ 21.9   │
+"""
```

Comment:
Is the newline added intended?

I prefer the output without newline, since the REPL will add one while rendering.

Reply:
You're right. I think I had it to separate the full printing of the TimeArray from the one with the omitted columns when `@show`ing.